### PR TITLE
Test for docker workflow

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Fixes https://github.com/inventree/InvenTree/issues/3996

<a href="https://gitpod.io/#https://github.com/inventree/InvenTree/pull/4014"></a>

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair it is getting even weirder. I have just run the "docker" workflow three times in this PR. The first two passed, and then the third run failed.

The point of failure is in the command `docker-compose run inventree-dev-server invoke test`

```
    CONFIG = config.load_config_data()
  File "/home/inventree/InvenTree/InvenTree/config.py", line 62, in load_config_data
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

This is running inside the docker container, where `pyyaml` has been installed as part of the docker image:

```
2022-12-02T09:40:45.9161973Z Collecting pyyaml>=6.0
2022-12-02T09:40:45.9323036Z   Downloading PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (661 kB)
```

```
2022-12-02T09:41:20.8528796Z Requirement already satisfied: pyyaml==6.0 in /usr/local/lib/python3.9/site-packages (from -r /home/inventree/requirements.txt (line 191)) (6.0)
```

So, we can see that it is installed within the container context. But then failing randomly?

**Comment by SchrodingersGat** [comment]
With the additional debug line above, when it *does load* it loads from:

```
YAML LOADED FROM: /usr/local/lib/python3.9/site-packages/yaml/__init__.py
```

**Comment by SchrodingersGat** [comment]
image

**Comment by matmair** [comment]
@SchrodingersGat this makes less and less sense... I do not really get how yaml could not be installed in the env

File: InvenTree/common/models.py [hunk 1]
```diff
@@ -1371,7 +1371,7 @@ def save(self, *args, **kwargs):
         'PLUGIN_ON_STARTUP': {
             'name': _('Check plugins on startup'),
             'description': _('Check that all plugins are installed on startup - enable in container environments'),
-            'default': settings.DOCKER,
+            'default': False,  # settings.DOCKER,
```
Comment (SchrodingersGat) [inline InvenTree/common/models.py:1374]: @matmair strangely, this change seems to have fixed the issue. Commenting out the line has resulted in 5/5 CI pass results in a row.

Comment (matmair) [reply InvenTree/common/models.py:1374]: Intresting! No idea why but let’s keep it that way. We can always enable plugins before tests.

**Review by matmair — APPROVED** [review]

**Comment by matmair** [comment]
Before merging you might want to remove the print statements introduced

**Pull request merged**